_includes/catalog-help-modal.html

```
@@ -98,55 +98,47 @@
 				data-fancybox="images" data-caption="Import your patterns"> -->
         <div style="text-align: center;">
           <video class="help-modal__video" preload="metadata" controls>
             <source src="{{ site.baseurl }}/assets/images/screens/pattern-import.mp4#t=0.01" type="video/mp4">
           </video>
-				<!-- <iframe width="98%" height="540" preload="metadata" src="{{ site.baseurl }}/assets/images/screens/pattern-import.mp4#t=0.01" alt="Import your patterns" 
+				<!-- <iframe width="98%" height="540" preload="metadata" src="{{ site.baseurl }}/assets/images/screens/pattern-import.mp4#t=0.01" alt="Import your patterns"
         frameborder="0"
         allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
         allowfullscreen	></iframe> -->
           </div>
 			</a>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </a> ]

(tag-pair)